src/ui/utils/mixpanel.ts

```diff
@@ -55,8 +55,14 @@ export async function trackMixpanelEvent(event: string, properties?: Dict) {
     console.log("🔴", event, properties);
   }
 
+  let namespace;
```

Comment:
maybe simplify to something like this?

```js
const namespaces = event.split(".");

mixpanel.track(
  event, 
  {
    ...properties, 
    namespace: nameSpaces.length ? nameSpace : null
  }
}
```
```

Reply:
as long as we are code golfing

`namespace: event.indexOf(".") ? event.split(".")[0] : ""`